addons/hypothermia/stringtable.xml

```diff
@@ -294,6 +294,7 @@
             <Italian>Impostazioni Ipotermia</Italian>
             <German>Hypothermieeinstellungen</German>
             <Japanese>低体温症設定</Japanese>
+            <Czech>Nastavení hypothermie</Czech>
```

Comment:
A tiny `h` managed to sneak it's way.

Reply:
No, mi nejde o to, aby jsi tam nějaké h přidával, ale spíše aby jsi odebral to, které tam máš navíc: hypot**H**ermie